content/en/docs/multi-center/modules/hosp/d_icd_diagnoses.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "d_icd_diagnoses"
+linktitle: "d_icd_diagnoses"
+weight: 6
+date: 2023-09-18
+description: >
+  Dimension table for *diagnoses_icd*; provides a description of ICD-9/ICD-10 billed diagnoses.
+---
+
+The *d_icd_diagnoses* table defines International Classification of Diseases (ICD) Version 9 and 10 codes for **diagnoses**. These codes are assigned at the end of the patient's stay and are used by the hospital to bill for care provided.
+
+### Links to
+
+* *diagnoses_icd* ON `icd_code` and `icd_version`
+
+## Table columns
+
+Name | Postgres data type
+---- | ----
+`icd_code` | CHAR(7) NOT NULL
+`icd_version` | INTEGER NOT NULL
+`long_title`  | VARCHAR(255)
+
+## Detailed Description
+
+### `icd_code`
+
+`icd_code` is the International Coding Definitions (ICD) code.
+
+### `icd_version`
+There are two versions for this coding system: version 9 (ICD-9) and version 10 (ICD-10). These can be differentiated using the `icd_version` column. [ICD-9](https://www.cms.gov/Medicare/Coding/ICD9ProviderDiagnosticCodes/codes) and [ICD-10](https://www.cms.gov/Medicare/Coding/ICD10/2019-ICD-10-CM.html) diagnosis codes are acquired from Centers for Medicare & Medicaid Services (CMS).
```

Comment:
all of the data for this subset should be ICD-10 - can't imagine NW using ICD-9 in 2020 onward - perhaps make it clear the description of ICD-9 is just for informational purposes?

Reply:
yes, added a clarifying statement to indicate ICD 9 codes are just for informational purposes, after mandate of ICD10 in 2015 based on CMS.